config/routes.rb

```diff
@@ -813,6 +813,9 @@
       patch '/nationalities' => 'application_forms/nationalities#update'
       get '/right-to-work-or-study' => 'application_forms/right_to_work_or_study#edit', as: :application_form_edit_right_to_work_or_study
       patch '/right-to-work-or-study' => 'application_forms/right_to_work_or_study#update'
+
+      get '/reinstate-offer/:application_choice_id' => 'application_forms/application_choices#reinstate_offer', as: :application_form_reinstate_offer
+      post '/reinstate-offer/:application_choice_id' => 'application_forms/application_choices#confirm_reinstate_offer'
```

Comment:
should be a `patch`

get - returns a resource
post - creates a rescouce/object
patch - updates an object